biome.json

```diff
@@ -1,15 +1,20 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.2.2/schema.json",
-  "organizeImports": {
-    "enabled": true
-  },
-  "linter": {
-    "enabled": true,
-    "rules": {
-      "recommended": true
-    }
-  },
-  "formatter": {
-    "formatWithErrors": true
-  }
-}
\ No newline at end of file
+	"$schema": "https://biomejs.dev/schemas/1.2.2/schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	},
+	"formatter": {
+		"enabled": true,
+		"formatWithErrors": true,
+		"indentStyle": "tab",
+		"indentWidth": 2,
+		"lineWidth": 120,
```

Comment:
Happy with 120? We were a bit back and forth between 100 and 120 :D

Reply:
Ah nice catch, was indeed supposed to be 100 :) Somehow managed to revert it back to 120 when extracting this into its own branch.

Reply:
On second thought, 100 looked better on a 1080p screen, but 120 feels better on any resolution above 1080p, so think I'll leave it at 120 for now. :)